# This message is a custom interface for get current state of movement action from robot
string state
# string name_service
uint8[16] uuid